controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -81,29 +81,30 @@ protected void startUp() throws InterruptedException {
                 switch (type) {
                     case HOST_ADDED:
                         // We need to do nothing when a new controller instance joins the cluster.
-                        log.info("Received controller cluster event: {} for host: {}", type, host);
+                        log.info("Received HOST_ADDED cluster event: {} for host: {}", type, host);
                         break;
                     case HOST_REMOVED:
-                        log.info("Received controller cluster event: {} for host: {}", type, host);
+                        log.info("Received HOST_REMOVED cluster event: {} for host: {}", type, host);
                         handleHostRemoved(host);
                         break;
                     case ERROR:
                         // This event should be due to ZK connection errors. If it is session lost error then
                         // ControllerServiceMain would handle it. Otherwise it is a fleeting error that can go
                         // away with retries, and hence we ignore it.
-                        log.info("Received error event when monitoring the controller host cluster, ignoring...");
+                        log.warn("Received error event from controller host {}", host);
                         break;
                 }
             }, executor);
 
+            // processes: set of controller process running at unique IP:PORT
             Supplier<Set<String>> processes = () -> {
                 try {
                     return cluster.getClusterMembers()
                             .stream()
                             .map(Host::getHostId)
                             .collect(Collectors.toSet());
                 } catch (ClusterException e) {
-                    log.error("error fetching cluster members {}", e);
+                    log.error("Error fetching cluster members {}", e);
```

Comment:
maybe warn?

Reply:
I think all connectivity issues, especially for crucial operations like this one could potentially cause a DUDL and so should be logged at `error` .
`warn` could be reserved for minor glitches in execution or connectivity failures that do not have any potential to cause DUDL.

Reply:
As discussed, logging this at warn, since its a retryable exception.